app/gui/view/component-browser/component-group/src/multi.rs

```diff
@@ -0,0 +1,255 @@
+//! Provides a multi component group wrapper, an abstraction used to propagate FRP events from
+//! multiple component groups.
+//!
+//! The wrapper can work with all types of component groups using [`Group`] enum. Currently we
+//! support one-column component groups and wide (multi-column) component groups.
+//!
+//! See [`Wrapper`] docs.
+
+use ensogl::prelude::*;
+
+use crate::entry;
+use crate::wide;
+use crate::View;
+
+use enso_frp as frp;
+use ensogl::data::OptVec;
+use ensogl::display::scene::Scene;
+
+
+
+// =============
+// === Group ===
+// =============
+
+/// One of the possible component group types.
+#[derive(Debug, Clone, CloneRef, From)]
+#[allow(missing_docs)]
+pub enum Group {
+    OneColumn(View),
+    Wide(wide::View),
+}
```

Comment:
I am almost sure I was writing in my previous review about it. These docs need to be expanded. What is "Wide"? I can't understand that when reading the code.